### ISSUE TITLE
Timeout on renewal acme.sh v3.0.1 Debian 8.7

### ISSUE THREAD

**Issue by merlyn12345** [acmesh-official/acme.sh#3658]
Hello

I was able to obtain a certificate for my domain www.naturundtext.de only once. The renewal via cronjob failed and my manual attempts all ran into a timout error.
acme.sh 3.0.1
The webserver is Apache 2.4.10 and it is listening on Port 80 as well as on 443. 
Linux is Debian 8.7
I tried this command: ./acme.sh --renew-all  --debug --log
------------------

Debug log output is:

[Mi 11. Aug 18:47:47 CEST 2021] Running cmd: renewAll
[Mi 11. Aug 18:47:47 CEST 2021] Using config home:/root/.acme.sh
[Mi 11. Aug 18:47:47 CEST 2021] default_acme_server
[Mi 11. Aug 18:47:47 CEST 2021] ACME_DIRECTORY='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:47 CEST 2021] _stopRenewOnError
[Mi 11. Aug 18:47:47 CEST 2021] _set_level='2'
[Mi 11. Aug 18:47:47 CEST 2021] di='/root/.acme.sh/www.naturundtext.de/'
[Mi 11. Aug 18:47:47 CEST 2021] d='www.naturundtext.de'
[Mi 11. Aug 18:47:47 CEST 2021] Using config home:/root/.acme.sh
[Mi 11. Aug 18:47:47 CEST 2021] ACME_DIRECTORY='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:47 CEST 2021] DOMAIN_PATH='/root/.acme.sh/www.naturundtext.de'
[Mi 11. Aug 18:47:47 CEST 2021] Renew: 'www.naturundtext.de'
[Mi 11. Aug 18:47:47 CEST 2021] Le_API='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:47 CEST 2021] Using config home:/root/.acme.sh
[Mi 11. Aug 18:47:47 CEST 2021] ACME_DIRECTORY='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:47 CEST 2021] _init api for server: https://acme.zerossl.com/v2/DV90
[Mi 11. Aug 18:47:47 CEST 2021] Retrying GET
[Mi 11. Aug 18:47:47 CEST 2021] GET
[Mi 11. Aug 18:47:47 CEST 2021] url='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:47 CEST 2021] timeout=
[Mi 11. Aug 18:47:47 CEST 2021] displayError='1'
[Mi 11. Aug 18:47:47 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:47 CEST 2021] ret='0'
[Mi 11. Aug 18:47:47 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_KEY_CHANGE='https://acme.zerossl.com/v2/DV90/keyChange'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_NEW_AUTHZ
[Mi 11. Aug 18:47:47 CEST 2021] ACME_NEW_ORDER='https://acme.zerossl.com/v2/DV90/newOrder'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_NEW_ACCOUNT='https://acme.zerossl.com/v2/DV90/newAccount'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_REVOKE_CERT='https://acme.zerossl.com/v2/DV90/revokeCert'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_AGREEMENT='https://secure.trust-provider.com/repository/docs/Legacy/20201020_Certificate_Subscriber_Agreement_v_2_4_click.pdf'
[Mi 11. Aug 18:47:47 CEST 2021] ACME_NEW_NONCE='https://acme.zerossl.com/v2/DV90/newNonce'
[Mi 11. Aug 18:47:47 CEST 2021] _main_domain='www.naturundtext.de'
[Mi 11. Aug 18:47:47 CEST 2021] _alt_domains='www.basilisken-presse.de,www.herpetoshop.de'
[Mi 11. Aug 18:47:47 CEST 2021] Using ACME_DIRECTORY: https://acme.zerossl.com/v2/DV90
[Mi 11. Aug 18:47:47 CEST 2021] _init api for server: https://acme.zerossl.com/v2/DV90
[Mi 11. Aug 18:47:47 CEST 2021] Le_NextRenewTime
[Mi 11. Aug 18:47:48 CEST 2021] Using CA: https://acme.zerossl.com/v2/DV90
[Mi 11. Aug 18:47:48 CEST 2021] _on_before_issue
[Mi 11. Aug 18:47:48 CEST 2021] _chk_main_domain='www.naturundtext.de'
[Mi 11. Aug 18:47:48 CEST 2021] _chk_alt_domains='www.basilisken-presse.de,www.herpetoshop.de'
[Mi 11. Aug 18:47:48 CEST 2021] Le_LocalAddress
[Mi 11. Aug 18:47:48 CEST 2021] d='www.naturundtext.de'
[Mi 11. Aug 18:47:48 CEST 2021] Check for domain='www.naturundtext.de'
[Mi 11. Aug 18:47:48 CEST 2021] _currentRoot='/var/www/live'
[Mi 11. Aug 18:47:48 CEST 2021] d='www.basilisken-presse.de'
[Mi 11. Aug 18:47:48 CEST 2021] Check for domain='www.basilisken-presse.de'
[Mi 11. Aug 18:47:48 CEST 2021] _currentRoot='apache'
[Mi 11. Aug 18:47:48 CEST 2021] d='www.herpetoshop.de'
[Mi 11. Aug 18:47:48 CEST 2021] Check for domain='www.herpetoshop.de'
[Mi 11. Aug 18:47:48 CEST 2021] _currentRoot='apache'
[Mi 11. Aug 18:47:48 CEST 2021] d
[Mi 11. Aug 18:47:48 CEST 2021] Using config home:/root/.acme.sh
[Mi 11. Aug 18:47:48 CEST 2021] ACME_DIRECTORY='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:48 CEST 2021] httpdconfname='apache2.conf'
[Mi 11. Aug 18:47:48 CEST 2021] httpdroot='/etc/apache2'
[Mi 11. Aug 18:47:48 CEST 2021] httpdconf='/etc/apache2/apache2.conf'
[Mi 11. Aug 18:47:48 CEST 2021] httpdconfname='apache2.conf'
[Mi 11. Aug 18:47:48 CEST 2021] Checking if there is an error in the apache config file before starting.
[Mi 11. Aug 18:47:48 CEST 2021] OK
[Mi 11. Aug 18:47:48 CEST 2021] Backup apache config file='/etc/apache2/apache2.conf'
[Mi 11. Aug 18:47:48 CEST 2021] JFYI, Config file /etc/apache2/apache2.conf is backuped to /root/.acme.sh/apache2.conf
[Mi 11. Aug 18:47:48 CEST 2021] In case there is an error that can not be restored automatically, you may try restore it yourself.
[Mi 11. Aug 18:47:48 CEST 2021] The backup file will be deleted on success, just forget it.
[Mi 11. Aug 18:47:48 CEST 2021] apacheVer='2.4.10'
[Mi 11. Aug 18:47:48 CEST 2021] _saved_account_key_hash is not changed, skip register account.
[Mi 11. Aug 18:47:48 CEST 2021] Read key length:
[Mi 11. Aug 18:47:48 CEST 2021] _createcsr
[Mi 11. Aug 18:47:48 CEST 2021] Multi domain='DNS:www.naturundtext.de,DNS:www.basilisken-presse.de,DNS:www.herpetoshop.de'
[Mi 11. Aug 18:47:48 CEST 2021] Getting domain auth token for each domain
[Mi 11. Aug 18:47:48 CEST 2021] d='www.basilisken-presse.de'
[Mi 11. Aug 18:47:48 CEST 2021] d='www.herpetoshop.de'
[Mi 11. Aug 18:47:48 CEST 2021] d
[Mi 11. Aug 18:47:48 CEST 2021] url='https://acme.zerossl.com/v2/DV90/newOrder'
[Mi 11. Aug 18:47:48 CEST 2021] payload='{"identifiers": [{"type":"dns","value":"www.naturundtext.de"},{"type":"dns","value":"www.basilisken-presse.de"},{"type":"dns","value":"www.herpetoshop.de"}]}'
[Mi 11. Aug 18:47:48 CEST 2021] RSA key
[Mi 11. Aug 18:47:48 CEST 2021] Retrying post
[Mi 11. Aug 18:47:48 CEST 2021] HEAD
[Mi 11. Aug 18:47:48 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/newNonce'
[Mi 11. Aug 18:47:48 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g  -I  '
[Mi 11. Aug 18:47:48 CEST 2021] _ret='0'
[Mi 11. Aug 18:47:48 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:49 CEST 2021] Retrying GET
[Mi 11. Aug 18:47:49 CEST 2021] GET
[Mi 11. Aug 18:47:49 CEST 2021] url='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:47:49 CEST 2021] timeout=
[Mi 11. Aug 18:47:49 CEST 2021] displayError='1'
[Mi 11. Aug 18:47:49 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:50 CEST 2021] ret='0'
[Mi 11. Aug 18:47:50 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:50 CEST 2021] Retrying GET
[Mi 11. Aug 18:47:50 CEST 2021] GET
[Mi 11. Aug 18:47:50 CEST 2021] url='https://acme.zerossl.com/v2/DV90/newNonce'
[Mi 11. Aug 18:47:50 CEST 2021] timeout=
[Mi 11. Aug 18:47:50 CEST 2021] displayError='1'
[Mi 11. Aug 18:47:50 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:50 CEST 2021] ret='0'
[Mi 11. Aug 18:47:50 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:50 CEST 2021] Retrying post
[Mi 11. Aug 18:47:50 CEST 2021] POST
[Mi 11. Aug 18:47:50 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/newOrder'
[Mi 11. Aug 18:47:50 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:51 CEST 2021] _ret='0'
[Mi 11. Aug 18:47:51 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:51 CEST 2021] code='201'
[Mi 11. Aug 18:47:51 CEST 2021] Le_LinkOrder='https://acme.zerossl.com/v2/DV90/order/Eh8Jxs5Ri_kCC-d_ZqWTNg'
[Mi 11. Aug 18:47:51 CEST 2021] Le_OrderFinalize='https://acme.zerossl.com/v2/DV90/order/Eh8Jxs5Ri_kCC-d_ZqWTNg/finalize'
[Mi 11. Aug 18:47:51 CEST 2021] url='https://acme.zerossl.com/v2/DV90/authz/W65ZAR97xnCZl6qms_bkiQ'
[Mi 11. Aug 18:47:51 CEST 2021] payload
[Mi 11. Aug 18:47:51 CEST 2021] Retrying post
[Mi 11. Aug 18:47:51 CEST 2021] POST
[Mi 11. Aug 18:47:51 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/authz/W65ZAR97xnCZl6qms_bkiQ'
[Mi 11. Aug 18:47:51 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:52 CEST 2021] _ret='0'
[Mi 11. Aug 18:47:52 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:52 CEST 2021] code='200'
[Mi 11. Aug 18:47:52 CEST 2021] url='https://acme.zerossl.com/v2/DV90/authz/j2BHnjF93bEfilLLc0hFiw'
[Mi 11. Aug 18:47:52 CEST 2021] payload
[Mi 11. Aug 18:47:52 CEST 2021] Retrying post
[Mi 11. Aug 18:47:52 CEST 2021] POST
[Mi 11. Aug 18:47:52 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/authz/j2BHnjF93bEfilLLc0hFiw'
[Mi 11. Aug 18:47:52 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:52 CEST 2021] _ret='0'
[Mi 11. Aug 18:47:52 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:52 CEST 2021] code='200'
[Mi 11. Aug 18:47:52 CEST 2021] url='https://acme.zerossl.com/v2/DV90/authz/89Na_SgF5RcxUv1zkH8rdA'
[Mi 11. Aug 18:47:52 CEST 2021] payload
[Mi 11. Aug 18:47:52 CEST 2021] Retrying post
[Mi 11. Aug 18:47:52 CEST 2021] POST
[Mi 11. Aug 18:47:52 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/authz/89Na_SgF5RcxUv1zkH8rdA'
[Mi 11. Aug 18:47:52 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:47:59 CEST 2021] _ret='0'
[Mi 11. Aug 18:47:59 CEST 2021] _hcode='0'
[Mi 11. Aug 18:47:59 CEST 2021] code='200'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.naturundtext.de'
[Mi 11. Aug 18:47:59 CEST 2021] Getting webroot for domain='www.naturundtext.de'
[Mi 11. Aug 18:47:59 CEST 2021] _w='/var/www/live'
[Mi 11. Aug 18:47:59 CEST 2021] _currentRoot='/var/www/live'
[Mi 11. Aug 18:47:59 CEST 2021] entry='"type":"http-01","url":"https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ","status":"pending","token":"fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8"'
[Mi 11. Aug 18:47:59 CEST 2021] token='fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8'
[Mi 11. Aug 18:47:59 CEST 2021] uri='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:47:59 CEST 2021] keyauthorization='fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0'
[Mi 11. Aug 18:47:59 CEST 2021] dvlist='www.naturundtext.de#fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ#http-01#/var/www/live'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.basilisken-presse.de'
[Mi 11. Aug 18:47:59 CEST 2021] Getting webroot for domain='www.basilisken-presse.de'
[Mi 11. Aug 18:47:59 CEST 2021] _w='apache'
[Mi 11. Aug 18:47:59 CEST 2021] _currentRoot='apache'
[Mi 11. Aug 18:47:59 CEST 2021] entry='"type":"http-01","url":"https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw","status":"pending","token":"NCvDZhQbq4FbQcUgskFbaOw9mpj4CJhpki6M6xMwbNI"'
[Mi 11. Aug 18:47:59 CEST 2021] token='NCvDZhQbq4FbQcUgskFbaOw9mpj4CJhpki6M6xMwbNI'
[Mi 11. Aug 18:47:59 CEST 2021] uri='https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw'
[Mi 11. Aug 18:47:59 CEST 2021] keyauthorization='NCvDZhQbq4FbQcUgskFbaOw9mpj4CJhpki6M6xMwbNI.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0'
[Mi 11. Aug 18:47:59 CEST 2021] dvlist='www.basilisken-presse.de#NCvDZhQbq4FbQcUgskFbaOw9mpj4CJhpki6M6xMwbNI.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw#http-01#apache'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.herpetoshop.de'
[Mi 11. Aug 18:47:59 CEST 2021] Getting webroot for domain='www.herpetoshop.de'
[Mi 11. Aug 18:47:59 CEST 2021] _w
[Mi 11. Aug 18:47:59 CEST 2021] _currentRoot='apache'
[Mi 11. Aug 18:47:59 CEST 2021] entry='"type":"http-01","url":"https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA","status":"pending","token":"ytEc4zZdDSvHnSP77nT6F3cK1hdwSLYqDnoNKfcg5VA"'
[Mi 11. Aug 18:47:59 CEST 2021] token='ytEc4zZdDSvHnSP77nT6F3cK1hdwSLYqDnoNKfcg5VA'
[Mi 11. Aug 18:47:59 CEST 2021] uri='https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA'
[Mi 11. Aug 18:47:59 CEST 2021] keyauthorization='ytEc4zZdDSvHnSP77nT6F3cK1hdwSLYqDnoNKfcg5VA.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0'
[Mi 11. Aug 18:47:59 CEST 2021] dvlist='www.herpetoshop.de#ytEc4zZdDSvHnSP77nT6F3cK1hdwSLYqDnoNKfcg5VA.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA#http-01#apache'
[Mi 11. Aug 18:47:59 CEST 2021] d
[Mi 11. Aug 18:47:59 CEST 2021] vlist='www.naturundtext.de#fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ#http-01#/var/www/live,www.basilisken-presse.de#NCvDZhQbq4FbQcUgskFbaOw9mpj4CJhpki6M6xMwbNI.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw#http-01#apache,www.herpetoshop.de#ytEc4zZdDSvHnSP77nT6F3cK1hdwSLYqDnoNKfcg5VA.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0#https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA#http-01#apache,'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.naturundtext.de'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.basilisken-presse.de'
[Mi 11. Aug 18:47:59 CEST 2021] d='www.herpetoshop.de'
[Mi 11. Aug 18:47:59 CEST 2021] ok, let's start to verify
[Mi 11. Aug 18:47:59 CEST 2021] Verifying: www.naturundtext.de
[Mi 11. Aug 18:47:59 CEST 2021] d='www.naturundtext.de'
[Mi 11. Aug 18:47:59 CEST 2021] keyauthorization='fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8.XMmGAH0tQX50uH3aVHoo4iGpO4arsxVHz0OH7E7LDQ0'
[Mi 11. Aug 18:47:59 CEST 2021] uri='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:47:59 CEST 2021] _currentRoot='/var/www/live'
[Mi 11. Aug 18:47:59 CEST 2021] wellknown_path='/var/www/live/.well-known/acme-challenge'
[Mi 11. Aug 18:47:59 CEST 2021] writing token:fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8 to /var/www/live/.well-known/acme-challenge/fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8
[Mi 11. Aug 18:47:59 CEST 2021] url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:47:59 CEST 2021] payload='{}'
[Mi 11. Aug 18:47:59 CEST 2021] Retrying post
[Mi 11. Aug 18:47:59 CEST 2021] POST
[Mi 11. Aug 18:47:59 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:47:59 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:48:00 CEST 2021] _ret='0'
[Mi 11. Aug 18:48:00 CEST 2021] _hcode='0'
[Mi 11. Aug 18:48:00 CEST 2021] code='200'
[Mi 11. Aug 18:48:00 CEST 2021] trigger validation code: 200
[Mi 11. Aug 18:48:00 CEST 2021] Processing, The CA is processing your order, please just wait. (1/30)
[Mi 11. Aug 18:48:00 CEST 2021] sleep 2 secs to verify again
[Mi 11. Aug 18:48:02 CEST 2021] checking
[Mi 11. Aug 18:48:02 CEST 2021] url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:48:02 CEST 2021] payload
[Mi 11. Aug 18:48:02 CEST 2021] Retrying post
[Mi 11. Aug 18:48:02 CEST 2021] POST
[Mi 11. Aug 18:48:02 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:48:02 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:48:03 CEST 2021] _ret='0'
[Mi 11. Aug 18:48:03 CEST 2021] _hcode='0'
[Mi 11. Aug 18:48:03 CEST 2021] code='200'
[Mi 11. Aug 18:48:03 CEST 2021] Processing, The CA is processing your order, please just wait. (2/30)
[Mi 11. Aug 18:48:03 CEST 2021] sleep 2 secs to verify again
[Mi 11. Aug 18:48:05 CEST 2021] checking

[ cut out 28 repetitions ]

[Mi 11. Aug 18:49:20 CEST 2021] Retrying post
[Mi 11. Aug 18:49:20 CEST 2021] POST
[Mi 11. Aug 18:49:20 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:49:20 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:49:21 CEST 2021] _ret='0'
[Mi 11. Aug 18:49:21 CEST 2021] _hcode='0'
[Mi 11. Aug 18:49:21 CEST 2021] code='200'
[Mi 11. Aug 18:49:21 CEST 2021] www.naturundtext.de:Timeout
[Mi 11. Aug 18:49:21 CEST 2021] Debugging, skip removing: /var/www/live/.well-known/acme-challenge/fPImGnN4XovWhdpWuwgVv_oW4MPka8rGFAx67u14Yg8
[Mi 11. Aug 18:49:21 CEST 2021] pid
[Mi 11. Aug 18:49:21 CEST 2021] Using config home:/root/.acme.sh
[Mi 11. Aug 18:49:21 CEST 2021] ACME_DIRECTORY='https://acme.zerossl.com/v2/DV90'
[Mi 11. Aug 18:49:21 CEST 2021] httpdconfname='apache2.conf'
[Mi 11. Aug 18:49:22 CEST 2021] httpdroot='/etc/apache2'
[Mi 11. Aug 18:49:22 CEST 2021] httpdconf='/etc/apache2/apache2.conf'
[Mi 11. Aug 18:49:22 CEST 2021] httpdconfname='apache2.conf'
[Mi 11. Aug 18:49:22 CEST 2021] Restored: /etc/apache2/apache2.conf.
[Mi 11. Aug 18:49:22 CEST 2021] Restored successfully.
[Mi 11. Aug 18:49:22 CEST 2021] No need to restore nginx, skip.
[Mi 11. Aug 18:49:22 CEST 2021] _clearupdns
[Mi 11. Aug 18:49:22 CEST 2021] dns_entries
[Mi 11. Aug 18:49:22 CEST 2021] skip dns.
[Mi 11. Aug 18:49:22 CEST 2021] _on_issue_err
[Mi 11. Aug 18:49:22 CEST 2021] Please check log file for more details: /root/.acme.sh/acme.sh.log
[Mi 11. Aug 18:49:22 CEST 2021] url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:49:22 CEST 2021] payload='{}'
[Mi 11. Aug 18:49:22 CEST 2021] Retrying post
[Mi 11. Aug 18:49:22 CEST 2021] POST
[Mi 11. Aug 18:49:22 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/-EkBMFfOxiLNEQ5xyvhOVQ'
[Mi 11. Aug 18:49:22 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:49:23 CEST 2021] _ret='0'
[Mi 11. Aug 18:49:23 CEST 2021] _hcode='0'
[Mi 11. Aug 18:49:23 CEST 2021] code='200'
[Mi 11. Aug 18:49:23 CEST 2021] url='https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw'
[Mi 11. Aug 18:49:23 CEST 2021] payload='{}'
[Mi 11. Aug 18:49:23 CEST 2021] Retrying post
[Mi 11. Aug 18:49:23 CEST 2021] POST
[Mi 11. Aug 18:49:23 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/Id9mJJFgzGPjRVOKhwK5Dw'
[Mi 11. Aug 18:49:23 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:49:23 CEST 2021] _ret='0'
[Mi 11. Aug 18:49:23 CEST 2021] _hcode='0'
[Mi 11. Aug 18:49:23 CEST 2021] code='200'
[Mi 11. Aug 18:49:23 CEST 2021] url='https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA'
[Mi 11. Aug 18:49:23 CEST 2021] payload='{}'
[Mi 11. Aug 18:49:23 CEST 2021] Retrying post
[Mi 11. Aug 18:49:23 CEST 2021] POST
[Mi 11. Aug 18:49:23 CEST 2021] _post_url='https://acme.zerossl.com/v2/DV90/chall/S30hRvApxp0QC8Uubx2lKA'
[Mi 11. Aug 18:49:23 CEST 2021] _CURL='curl --silent --dump-header /root/.acme.sh/http.header  -L  -g '
[Mi 11. Aug 18:49:24 CEST 2021] _ret='0'
[Mi 11. Aug 18:49:24 CEST 2021] _hcode='0'
[Mi 11. Aug 18:49:24 CEST 2021] code='200'
[Mi 11. Aug 18:49:24 CEST 2021] socat doesn't exist.
[Mi 11. Aug 18:49:24 CEST 2021] Diagnosis versions:
openssl:openssl
OpenSSL 1.0.1t  3 May 2016
apache:
Server version: Apache/2.4.10 (Debian)
Server built:   Feb 24 2017 18:40:28
Server's Module Magic Number: 20120211:37
Server loaded:  APR 1.5.1, APR-UTIL 1.5.4
Compiled using: APR 1.5.1, APR-UTIL 1.5.4
Architecture:   64-bit
Server MPM:     prefork
  threaded:     no
    forked:     yes (variable process count)
Server compiled with....
 -D APR_HAS_SENDFILE
 -D APR_HAS_MMAP
 -D APR_HAVE_IPV6 (IPv4-mapped addresses enabled)
 -D APR_USE_SYSVSEM_SERIALIZE
 -D APR_USE_PTHREAD_SERIALIZE
 -D SINGLE_LISTEN_UNSERIALIZED_ACCEPT
 -D APR_HAS_OTHER_CHILD
 -D AP_HAVE_RELIABLE_PIPED_LOGS
 -D DYNAMIC_MODULE_LIMIT=256
 -D HTTPD_ROOT="/etc/apache2"
 -D SUEXEC_BIN="/usr/lib/apache2/suexec"
 -D DEFAULT_PIDLOG="/var/run/apache2.pid"
 -D DEFAULT_SCOREBOARD="logs/apache_runtime_status"
 -D DEFAULT_ERRORLOG="logs/error_log"
 -D AP_TYPES_CONFIG_FILE="mime.types"
 -D SERVER_CONFIG_FILE="apache2.conf"
nginx:
nginx doesn't exist.
socat:
[Mi 11. Aug 18:49:24 CEST 2021] Return code: 1
[Mi 11. Aug 18:49:24 CEST 2021] Error renew www.naturundtext.de.
[Mi 11. Aug 18:49:24 CEST 2021] _error_level='1'
[Mi 11. Aug 18:49:24 CEST 2021] _set_level='2'
-----------------
I also tried to get a new certificate via 
./acme.sh --issue -d www.naturundtext.de -d www.basilisken-presse.de -d www.herpetoshop.de --webroot /var/www/live --apache --force
but that failed too.

I am stuck here. Any help appreciated.

Cheers Georg

**Comment by Neilpang** [comment]
remove ` --webroot /var/www/live `

**Comment by wbeckle** [comment]
check that you don't have an unconditional permanent redirection to https in your config.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
